invalid_ebml/readme.md

```diff
@@ -0,0 +1,133 @@
+# valid files
+
+## reference.mkv
+
+A reference file was produced via:
+
+```
+ffmpeg -f lavfi -i mandelbrot=s=320x280:r=25 -f lavfi -i sine -t 0.5 -c:v ffv1 -c:a flac reference.mkv
+mkclean reference.mkv
+```
+
+# invalid files
+
+## no_DocType.mkv
+
+The EBML Header contains no mandated DocType Element.
+
+[no_DocType.mkv](no_DocType.mkv)
+
+## docTypeReadVersion_greaterthan_docTypeVersion.mkv
+
+The docTypeReadVersion MUST be less than docTypeVersion but here it isn't.
+
+[docTypeReadVersion_greaterthan_docTypeVersion.mkv](docTypeReadVersion_greaterthan_docTypeVersion.mkv)
+
+## EBMLReadVersion_greaterthan_EBMLVersion.mkv
+
+The EBMLReadVersion MUST be less than EBMLVersion but here it isn't.
+
+[EBMLReadVersion_greaterthan_EBMLVersion.mkv](EBMLReadVersion_greaterthan_EBMLVersion.mkv)
+
+## currently_undefined_EBMLVersion.mkv
+
+The EBMLVersion is restricted to a range of defined values. At the time of this writing, only EBMLVersion=1 is defined. This value has an EBMLVersion of 255.
```

Comment:
I'm confused on the issue with this file. Here is the start of the output of mkvinfo:

```
mkvinfo /Users/drice/github/matroska-test-files/invalid_ebml/currently_undefined_EBMLVersion.mkv 
+ EBML head
|+ EBML version: 255
|+ EBML read version: 1
|+ EBML maximum ID length: 4
|+ EBML maximum size length: 8
|+ Doc type: matroska
|+ Doc type version: 4
|+ Doc type read version: 2
```

Presently EBMLVersion has a range="1" meaning that if the EBMLVersion is there and not an Empty Element than the value (until EBML 2 is defined) must be "1".